util/deltautil/apply.go

```diff
@@ -0,0 +1,178 @@
+package deltautil
+
+import (
+	"fmt"
+
+	pb "github.com/earthly/cloud-api/logstream"
+)
+
+// Version is the version of the deltautil package.
+const Version = 2
```

Comment:
Because we called the existing non-streaming version currently in production as v1. We could also call this one version one, since they don't clash in any way (they're completely different implementations).